config/config.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"github.com/joho/godotenv"
+	"log"
+)
+
+// EnvVariablesPostgres holds the environment variables for PostgreSQL connection
+type EnvVariablesPostgres struct {
+	PostgresUser     string
+	PostgresPassword string
+	PostgresDBName   string
+}
+
+// ConnVarPostgres is a global variable that holds the PostgreSQL connection variables
+var ConnVarPostgres EnvVariablesPostgres
+
+// DBConnStr is the connection string for the PostgreSQL database
+var DBConnStr string
+
+// init function initialises the environment variables from the .env file
+func init() {
```

Comment:
using init here means that these variables must be set during unit tests as well. how do you plan to have tests for components that rely on config?